tooling/release_download_test.sh

```diff
@@ -344,7 +344,7 @@ verify_gcc_version() {
   # shellcheck disable=SC2166
   [ "${MAJOR_VERSION}" = "8" -o "${MAJOR_VERSION}" = "11" ] && expected_gcc=7.5.0
   [ "${MAJOR_VERSION}" = "17" ] && expected_gcc=10.3.0
-  [ "${MAJOR_VERSION}" -ge 20 ] && expected_gcc=11.2.0
+  [ "${MAJOR_VERSION}" -ge 20 ] && expected_gcc=11.3.0
```

Comment:
That check is only invoked on the platform that you run the script on (it's downloading the version for the current platform and examining it). There is no expectation that it will run on anything other than Linux/x64 or Linux/aarch64 which is what it's been tested on.
For the checks for each platform's download and SBoM it uses the checks in `validateSBOMcontent.sh` which handles the Alpine version along with all the others.

If someone wishes to enable it to run on Alpine or other architectures they are welcome to do so, but that's not the scope of this PR, and this does not make that situation worse than it has been up to now ;-)

Reply:
(I initially had the same thought as you but regarding riscv64 which doesn't have a devkit yet)